src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const mans = century
+    ? people.filter(person => (
+      person.sex === 'm'
+      && Math.ceil(person.died / 100) === century
+    ))
+    : people.filter(person => person.sex === 'm');
```

Comment:
there is one more way to write this
```suggestion
  const mans = people.filter(person => (century)
    ? person.sex === 'm'
      && Math.ceil(person.died / 100) === century
     :person.sex === 'm';
  )
```